flyte_local.yaml

```diff
@@ -7,7 +7,7 @@
 # flyte start --config flyte_local.yaml
 propeller:
   rawoutput-prefix: "s3://my-s3-bucket/test/"
-  kube-config: "$HOME/.flyte/k3s/k3s.yaml"
+  kube-config: "$HOME/.flyte/sandbox/kubeconfig"
```

Comment:
Refer to the how the current flytectl works: https://github.com/flyteorg/flytectl/blob/0cb2bfaf436a09d58c1ae9c1389eacb6eab8a856/pkg/docker/docker_util.go#L30

Reply:
Is it this variable or SandboxKubeconfig that's used in `flytectl demo`?  Let's keep whatever is used for `demo` not `sandbox`.

Reply:
otherwise looks good.

Reply:
It is also used for `flytectl demo`.  

[startDemoCluster](https://github.com/flyteorg/flytectl/blob/ac3aa739aa7a499179a298715641aaf9ca836193/cmd/demo/start.go#L89) -> [sandbox.StartDemoCluster](https://github.com/flyteorg/flytectl/blob/ac3aa739aa7a499179a298715641aaf9ca836193/pkg/sandbox/start.go#L389C6-L389C22) -> [StartCluster](https://github.com/flyteorg/flytectl/blob/ac3aa739aa7a499179a298715641aaf9ca836193/pkg/sandbox/start.go#L251C6-L251C18) -> [UpdateLocalKubeContext](https://github.com/flyteorg/flytectl/blob/ac3aa739aa7a499179a298715641aaf9ca836193/pkg/sandbox/start.go#L326). 

UpdateLocalKubeContext copies the kubeconfig to [$HOME/.flyte/sandbox/kubeconfig ](https://github.com/flyteorg/flytectl/blob/0cb2bfaf436a09d58c1ae9c1389eacb6eab8a856/pkg/docker/docker_util.go#L30)